utils.go

```diff
@@ -199,6 +208,13 @@ func mustBeInteger(what interface{}) *int {
 			return nil
 		}
 
+	} else if isInteger(what) {
```

Comment:
i think you could get the same effect without needing to involve string parsing by just changing `isInteger` to return `(int64, bool)` and in the switch statement therecan then call `return int64(what), true`.

Reply:
Oh, I don't think you can do that because it still considers the number an interface{} because it can't tell which type of int it is since the `case` is for multiple types.

Reply:
Ahh yeah